extensions/ssh-manager/src/create.tsx

```diff
@@ -0,0 +1,36 @@
+import { ActionPanel, Form, showHUD, SubmitFormAction, useNavigation, randomId } from "@raycast/api";
+import { ISSHConnection } from "./types";
+import { getConnections, saveConnections } from "./storage.api";
+
+export default function Main() {
+  const { pop } = useNavigation();
+
+  async function saveConnection(connection: ISSHConnection) {
+    const existingConnections = await getConnections();
+    existingConnections.push({ ...connection, id: randomId() });
+
+    await saveConnections(existingConnections);
+    await showHUD("Saved connection ✅");
+
+    pop();
+  }
+
+  return (
+    <Form
+      actions={
+        <ActionPanel>
+          <SubmitFormAction title="Save" onSubmit={(values) => saveConnection(values as ISSHConnection)} />
+        </ActionPanel>
+      }
+    >
+      <Form.TextField id="name" title="Connection Name" />
+      <Form.TextField id="address" title="Server Address" placeholder={"A resolvable DNS name or IP"} />
+      <Form.TextField id="user" title="Username" placeholder={"A username to authenticate with"} />
+      <Form.TextField
+        id="key"
```

Comment:
```suggestion
        id="sshKey"
```
Looks like it has the wrong name and won't work without this